### DISCUSSION TITLE
Multiple like query on the same column

### DISCUSSION THREAD

**Discussion by prince-raza** [ujjwalguptaofficial/JsStore#266] [Q&A]
Hi, I was just wondering is 
"SELECT * FROM table WHERE name like : `%${input}%` AND name like : `%${input2}%` ..." or
"SELECT * FROM table WHERE name like : `%${input}%` OR name like : `%${input2}%` ..." 
Possible? Thank you 🙇

**Comment by ujjwalguptaofficial** [comment]
yep possible,
please have a look at these examples query - https://jsstore.net/tutorial/adv-sql-example/

let me know if you are still confused.
